pkg/reconciler/ingress/resources/gateway.go

```diff
@@ -146,12 +147,12 @@ func MakeExternalIngressGateways(ctx context.Context, ing *v1alpha1.Ingress, ser
 
 // MakeWildcardTLSGateways creates gateways that only contain TLS server with wildcard hosts based on the wildcard secret information.
 // For each public ingress service, we will create a list of Gateways. Each Gateway of the list corresponds to a wildcard cert secret.
-func MakeWildcardTLSGateways(ctx context.Context, originWildcardSecrets map[string]*corev1.Secret,
+func MakeWildcardTLSGateways(ctx context.Context, ing *v1alpha1.Ingress, originWildcardSecrets map[string]*corev1.Secret,
```

Comment:
Since we are creating the gateways based on the ingress info as well, we probably need to update the description to reflect that.

Reply:
lmk if it's more clear

Reply:
Maybe rephrase to "Gateways generated are based on the related ingress being reconciled".